src/pages/joinGameLobby/[id].js

```diff
@@ -142,11 +186,12 @@ export default function Lobby() {
             onChange={gameLobbyChangeHandler}
             value={gameLobbyText}
           ></input>
+
           <button
             className="startButton"
             disabled={buttonDisabled}
             style={buttonStyle2}
-            onClick={assignRoles(fakePlayers)}
+            onClick={() => assignRoles(fakePlayers)}
```

Comment:
can you create a game and redirect player to that game screen on this click? at this point you should have everything you need to send on req.body: { gameID(String), users: [{userName, rank: 1, role}], phase: "day"}
and to navigate to the correct game page(...:3000/game/${gameID}